Raider coat fixes

juxjux9930 opened the pull request:
🆑 
tweak: The pirate coat suit that raiders get no longer lies about being space-proof.
/ 🆑 

We had two different coats using the same sprite with the same description. I've merged them into one under the path that doesn't try to pretend it's a spacesuit.

juxtaposed9930 commented:
You are pain, funny moustache main Travis

Hubblenaut reviewed (changes requested):
Since it's not a space suit and opposed to what it says in its autogenerated codex entry does actually *not* protect from space, the actual issue here seems to be that it should have never been defined as a subtype of `suit/space` in the first place.

Redefining it as `/obj/item/clothing/suit/pirate` would probably be the more sensible solution here instead of undoing the space suit flags of its parent.

juxtaposed9930 commented:
Last time I checked, that item path already belongs to something.

juxtaposed9930 commented:
I was correct! That path belongs too.... the same item? Wait, no, it's just a coat with the same name and icons!

_ahhhhh_

Right, I'll just 
yeah

juxtaposed9930 commented:
Right, the funny moustache man won't be made happy.

SierraKomodo commented:
Rebase to fix travis.

Spookerton commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -33,7 +33,23 @@
 	icon_state = "pirate"
-	body_parts_covered = UPPER_TORSO|ARMS
+	w_class = ITEM_SIZE_NORMAL
+	allowed = list(
+					/obj/item/weapon/gun,/obj/item/ammo_magazine,
+					/obj/item/ammo_casing,/obj/item/weapon/melee/baton,
+					/obj/item/weapon/handcuffs,/obj/item/weapon/tank/emergency
+					)
```
```suggestion
	allowed = list(
		/obj/item/weapon/gun,
		/obj/item/ammo_magazine,
		/obj/item/ammo_casing,
		/obj/item/weapon/melee/baton,
		/obj/item/weapon/handcuffs,
		/obj/item/weapon/tank/emergency
	)
```

Spookerton commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -33,7 +33,23 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SMALL,
+		rad = ARMOR_RAD_MINOR
+		)
```
```suggestion
	)
```

Spookerton commented on code/modules/clothing/spacesuits/miscellaneous.dm:
```diff
@@ -103,4 +62,5 @@
-	tinted = FALSE
\ No newline at end of file
+	tinted = FALSE
+	
```
```suggestion
```

Spookerton commented on code/modules/clothing/spacesuits/miscellaneous.dm:
```diff
@@ -8,12 +8,12 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SHIELDED,
 		rad = ARMOR_RAD_RESISTANT
 		)
```
```suggestion
	)
```

Spookerton commented on code/modules/clothing/head/helmet.dm:
```diff
@@ -218,4 +218,23 @@
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_SMALL,
+		rad = ARMOR_RAD_MINOR
+		)
```
```suggestion
	)
```

juxjux9930 committed:
Raider coat doesn't make you nude


Update code/modules/clothing/suits/miscellaneous.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/modules/clothing/suits/miscellaneous.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/modules/clothing/spacesuits/miscellaneous.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/modules/clothing/spacesuits/miscellaneous.dm

Co-authored-by: Spookerton <spkrtn@pm.me>
Update code/modules/clothing/head/helmet.dm

Co-authored-by: Spookerton <spkrtn@pm.me>

juxtaposed9930 commented:
Suggestions taken and squasheded
